### ISSUE TITLE
Unable to reproduce your result

### ISSUE THREAD

**Issue by s-ryosky** [CVMI-Lab/ST3D#7]
I have used the following config to pre-train a model.
https://github.com/CVMI-Lab/ST3D/blob/73dda8c131b960f6970ffd49d44d0f15f6f60f00/tools/cfgs/da-waymo-kitti_models/secondiou/secondiou_old_anchor_sn.yaml

The best evaluation results of the pre-trained model on WOD (23th epoch) were as follows:
OBJECT_TYPE_TYPE_VEHICLE_LEVEL_1/AP: 0.4767 
OBJECT_TYPE_TYPE_VEHICLE_LEVEL_1/APH: 0.4701 
OBJECT_TYPE_TYPE_VEHICLE_LEVEL_2/AP: 0.4155 
OBJECT_TYPE_TYPE_VEHICLE_LEVEL_2/APH: 0.4096

Then, I have used the following config to self-train the model.
https://github.com/CVMI-Lab/ST3D/blob/73dda8c131b960f6970ffd49d44d0f15f6f60f00/tools/cfgs/da-waymo-kitti_models/secondiou_st3d/secondiou_st3d.yaml

The best evaluation results of the self-trained model on KITTI (11th epoch) were as follows:
Car AP@0.70, 0.70, 0.70:
bbox AP:88.8405, 84.5192, 84.6117
bev  AP:87.7364, 78.1648, 77.4027
3d   AP:68.8041, 60.3439, 59.5532
aos  AP:88.73, 84.20, 84.05
Car AP_R40@0.70, 0.70, 0.70:
bbox AP:93.6732, 85.0615, 84.7418
bev  AP:90.1386, 81.8106, 81.0929
3d   AP:68.9595, 59.3669, 57.4259
aos  AP:93.54, 84.74, 84.18
Car AP@0.70, 0.50, 0.50:
bbox AP:88.8405, 84.5192, 84.6117
bev  AP:95.3801, 88.1350, 87.7074
3d   AP:95.2582, 87.8998, 87.4439
aos  AP:88.73, 84.20, 84.05
Car AP_R40@0.70, 0.50, 0.50:
bbox AP:93.6732, 85.0615, 84.7418
bev  AP:97.4356, 92.7666, 90.8911
3d   AP:97.2213, 90.8523, 90.5397
aos  AP:93.54, 84.74, 84.18

**Comment by jihanyang** [comment]
Hello, other researcher have reproduced this work with the release code (see #5). Notice that you are supposed to use kitti metric to evaluate models in the pretraining stage. Also, other factors like batch size per gpu are matters for the final results.
